test/client.js

```diff
@@ -1076,6 +1090,56 @@ describe('client API', function() {
     });
   });
 
+  describe('Multiple output proposals', function() {
+    it('should create, get, sign, and broadcast proposal', function(done) {
+      helpers.createAndJoinWallet(clients, 1, 1, function(w) {
+        clients[0].createAddress(function(err, x0) {
+          should.not.exist(err);
+          should.exist(x0.address);
+          blockchainExplorerMock.setUtxo(x0, 1, 1);
+          var opts = {
+            type: 'multiple_outputs',
+            message: 'hello',
+            outputs: [
+              {
+                amount: 10000,
+                toAddress: 'n2TBMPzPECGUfcT2EByiTJ12TPZkhN2mN5',
+                message: 'world',
+              },
+              {
+                amount: 20000,
+                toAddress: 'n2TBMPzPECGUfcT2EByiTJ12TPZkhN2mN5',
+                message: 'nakamotosan',
+              }
+            ]
+          };
+          clients[0].sendTxProposal(opts, function(err, x) {
+            should.not.exist(err);
+            clients[0].getTx(x.id, function(err, x2) {
+              should.not.exist(err);
+              x2.creatorName.should.equal('creator');
+              x2.message.should.equal('hello');
+              x2.fee.should.equal(10000);
+              x2.outputs[0].toAddress.should.equal('n2TBMPzPECGUfcT2EByiTJ12TPZkhN2mN5');
+              x2.outputs[0].amount.should.equal(10000);
+              x2.outputs[0].message.should.equal('world');
```

Comment:
Do you think it would improve the test if some checks are performed on the second output too?